tests/manage/z_cluster/test_validate_ceph_config_values.py

```diff
@@ -0,0 +1,50 @@
+import logging
+import pytest
+import collections
+
+
+from ocs_ci.framework.testlib import ManageTest, tier1
+from ocs_ci.framework import config
+from ocs_ci.ocs.ocp import OCP
+from ocs_ci.ocs import constants, defaults
+
+
+log = logging.getLogger(__name__)
+
+
+@tier1
+@pytest.mark.skipif(
+    config.DEPLOYMENT["ceph_debug"],
+    reason="Ceph was configured with customized values by ocs-ci so there is point in validating its config values",
+)
+class TestValidateCephConfigValues(ManageTest):
+    """
+    Test class for Ceph config values validation
+    """
+
+    def test_validate_ceph_config_values(self):
+        """
+        Test case for comparing the cluster's config values of
+        Ceph with the static set of configuration saved in ocs-ci
+
+        """
+        cm_obj = OCP(
+            kind="configmap",
+            namespace=defaults.ROOK_CLUSTER_NAMESPACE,
+            resource_name=constants.ROOK_CONFIG_OVERRIDE_CONFIGMAP,
+        )
+        config_data = cm_obj.get().get("data").get("config")
```

Comment:
Just use `cm_obj.get()["data"]["config"]`, since the code here:
- will produce meaningless exception *None doesn't have get() method* in case of a failure
- doesn't react or can't react on the failure in any other way than to fail the test

Reply:
done